schemas/v1.0/mockresponseplugin.schema.json

```diff
@@ -41,33 +41,40 @@
             ]
           },
           "response": {
-            "body": {
-              "type": [
-                "object",
-                "string"
-              ]
-            },
-            "code": {
-              "type": "integer"
-            },
-            "headers": {
-              "type": "array",
-              "items": {
-                "type": "object",
-                "properties": {
-                  "name": {
-                    "type": "string"
-                  },
-                  "value": {
-                    "type": "string"
-                  }
-                },
-                "required": [
-                  "name",
-                  "value"
+            "type": "object",
+            "properties": {
+              "body": {
+                "type": [
+                  "object",
+                  "string"
                 ]
+              },
+              "statusCode": {
+                "type": "integer"
+              },
+              "headers": {
+                "type": "array",
+                "items": {
+                  "type": "object",
+                  "properties": {
+                    "name": {
+                      "type": "string"
+                    },
+                    "value": {
+                      "type": "string"
+                    }
+                  },
+                  "required": [
+                    "name",
+                    "value"
+                  ]
+                }
               }
-            }
+            },
+            "required": [
```

Comment:
None of the properties are required. In theory, all of them could be empty and you'd get a 200 OK response. So let's remove it.